frontend/src/metabase-lib/Question.ts

```diff
@@ -46,7 +43,7 @@ import {
 } from "metabase-types/types/Visualization";
 import { DependentMetadataItem } from "metabase-types/types/Query";
 import { utf8_to_b64url } from "metabase/lib/encoding";
-import { CollectionId } from "metabase-types/api";
+import { CollectionId, Parameter as ParameterObject } from "metabase-types/api";
```

Comment:
I know this is an import, but I don't think we should always rename a named import like this. There are a couple of places that rename `Parameter` to `ParameterObject` and I don't know why we need to rename it, can't it just be `Parameter`?